src/stripe.android.ts

```diff
@@ -50,6 +51,53 @@ export class Stripe {
     );
   }
 
+  createSource(card: CardCommon, cb: (error: Error, source: Source) => void): void {
+    if (!card) {
+      if (typeof cb === 'function') {
+        cb(new Error('Invalid card'), null);
+      }
+      return;
+    }
+    const cardSourceParams = com.stripe.android.model.SourceParams.createCardParams(card.native)
+
+    try {
+      this._stripe.createSource(
+        cardSourceParams, 
+        new com.stripe.android.SourceCallback({
+          onSuccess: function(source) {
+              if (typeof cb === 'function') {
+                const newSource: Source = {
+                  id: source.getId(),
+                  amount: source.getAmount(),
+                  card: card,
+                  client_secret: source.getClientSecret(),
+                  created: source.getCreated(),
+                  currency: source.getCurrency(),
+                  flow: source.getFlow(),
+                  livemode: source.isLiveMode(),
+                  metadata: source.getMetaData(),
+                  object: source.getSourceTypeData(),
+                  owner: source.getOwner(),
```

Comment:
This needs to be explicitly mapped. It's a pain. :-)  Is `owner` a field you need on the client?